sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentField.java

```diff
@@ -15,7 +15,7 @@
  * An object representing the content and location of a field value.
  */
 @Immutable
-public final class DocumentField {
+public final class DocumentField extends TypedDocumentField {
```

Comment:
This should be 
```suggestion
public final class DocumentField extends TypedDocumentField<Object> {
```

Reply:
Also, looking further down, I don't think you need fields for `valueString`, `valueDate`, etc - just have a `value` and cast it in the getValueAsString methods, etc.